tests/GrabyTest.php

```diff
@@ -17,7 +17,7 @@ public function testConstructDefault()
     {
         $graby = new Graby(['debug' => true]);
 
-        $this->assertTrue($graby->getConfig('debug'));
```

Comment:
You should keep this one too

Reply:
https://github.com/j0k3r/graby/pull/184/commits/df26846ea7690e95bf00a445d0be095e799884eb